core/drivers/remoteproc/stm32_remoteproc.c

```diff
@@ -381,6 +384,17 @@ static TEE_Result stm32_rproc_probe(const void *fdt, int node,
 			goto err;
 	}
 
+#ifdef CFG_STM32MP15
+	if (!stm32_rcc_is_secure()) {
+		if (IS_ENABLED(CFG_INSECURE))
+			IMSG("WARNING: insecure rproc support regarding RCC hardening");
+		else
+			panic("RCC secure hardening issue");
```

Comment:
Can keep the trace above and replace WARNING: with ERROR:

Reply:
ok